lib/svg-preview-view.js

```diff
@@ -27,6 +27,17 @@ class SvgPreviewView extends View {
             this.text('transparent')
           })
         })
+        this.div({ class: 'image-controls-zoom-group' }, () => {
+          this.a({ outlet: 'zoomInButton', class: 'image-controls-zoom-in' }, () => {
+            this.text('+')
+          })
+          this.a({ outlet: 'zoomOutButton', class: 'image-controls-zoom-out' }, () => {
+            this.text('-')
+          })
+          this.a({ outlet: 'zoomResetButton', class: 'image-controls-zoom-reset' }, () => {
+            this.text('1:1')
+          })
+        })
```

Comment:
Aha, I am sorry I missed that part. I didn't look closely enough. Just starting with Atom UI stuff. Thanks for helping through this.